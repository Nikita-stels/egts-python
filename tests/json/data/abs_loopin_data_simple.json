{
  "transport": {
    "pt": 1
  },
  "service": {
    "sdr": [
      {
        "rfl": {
          "ssod": 0,
          "rsod": 0
        },
        "tm": "0x0fd90000",
        "sst": 2,
        "rst": 2,
        "rd": [
          {
            "srt": 26,
            "srd": {
              "lin": 300,
              "lis": 1
            }
          }
        ]
      }
    ]
  }
}
