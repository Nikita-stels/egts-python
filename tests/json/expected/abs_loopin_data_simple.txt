EGTS protocol validator v.1.0  Copyright JSC "Navigation-information systems", 2011
------------------------------------------------------------------------------------
File: 'D:\PycharmProjects\egts_protocol\tests\json\generated\tmp_msg.bin'
Packet data: 
 0100000B0010000100011F05000100040000D90F02021A0200C112EBCF


EGTS Transport Layer:
---------------------
  Validating result   - 0 (OK)

  Protocol Version    - 1
  Security Key ID     - 0
  Flags               - 00000000b (0x00)
       Prefix         - 00
       Route          -   0
       Encryption Alg -    00
       Compression    -      0
       Priority       -       00 (the highest)
  Header Length       - 11
  Header Encoding     - 0
  Frame Data Length   - 16
  Packet ID           - 1
  No route info       -
  Header Check Sum    - 0x1F

EGTS Service Layer:
---------------------
  Validating result   - 0 (OK)

  Packet Type         - EGTS_PT_APPDATA
  Service Layer CS    - 0xCFEB

    Service Layer Record:
    ---------------------
    Validating Result    - 0 (OK)

    Record Length               - 5
    Record Number               - 1
    Record flags                -     00000100b (0x04)
        Sourse Service On Device    - 0
        Recipient Service On Device -  0
        Group Flag                  -   0
        Record Processing Priority  -    00 (the highest)
        Time Field Exists           -      1
        Event ID Field Exists       -       0
        Object ID Field Exists      -        0
    Time                        - 0x0FD90000 (05.06.2018 07:25:52 GMT)
    Source Service Type         - 2 (EGTS_TELEDATA_SERVICE) 
    Recipient Service Type      - 2 (EGTS_TELEDATA_SERVICE) 

       Subrecord Data:
       ------------------
       Validating Result   - 150 (Unknown service)

       Subrecord Type      - 26 (unspecified)
       Subrecord Length    - 2


-------------------------------------------------------------
Press any key to exit...
